modules/auxiliary/admin/kerberos/forge_ticket.rb

```diff
@@ -108,8 +108,6 @@ def forge_ccache(sname:, flags:)
   end
 
   def forge_silver
-    raise Msf::OptionValidateError, 'SPN' if datastore['SPN'].blank?
```

Comment:
I think a more correct fox for this is
`raise Msf::OptionValidateError, ['SPN'] if datastore['SPN'].blank?`

I'm not sure how this got in as is that's my bad, but I do think we want to raise a validiation error if you try to create a silver ticket without an SPN?

Reply:
An error is still raised, but now it's from the `fail_with(Failure::BadConfig, ...` in `#validate_spn!`. Does that work, or should it be `Msf::OptionValidateError` specifically? What I liked about the `Failure::BadConfig` is that it's what the other `#validate_` methods use and it also included details as to how it should be formatted.

This is the error now:
```
msf6 auxiliary(admin/kerberos/forge_ticket) >  run

[-] Auxiliary aborted due to failure: bad-config: Invalid SPN, must be in the format <service class>/<host><realm>:<port>/<service name>. Ex: cifs/ws.msflab.local@MSFLAB.LOCAL
[*] Auxiliary module execution completed
msf6 auxiliary(admin/kerberos/forge_ticket) >
```

Reply:
uhhh yea actually that's fine, at the time I wanted there to be a different error message for not setting it vs setting it incorrectly but you're right might as well just output the better message